fourmolu/info.yaml

```diff
@@ -0,0 +1,31 @@
+---
+name: fourmolu
```

Comment:
I think we should release this at first with `enabled: false`.

So far, I've elected not to make a choice between the "opinionated" Haskell formatters for what should be on by default. I think I should revisit that and pick one, but I'm not sure it would be (f)ormulu, given just how opinionated it is.

Is that alright, or do you want to make a case for on-by-default?

Reply:
That's fine, I'll disable it.